src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -95,30 +95,34 @@ func main() {
 }
 
 func _main() {
-	err := doMain()
-	if err != nil {
+	if err := doMain(); err != nil {
 		log.Error(err)
 	}
 	log.Debug("Lantern stopped")
-	logging.Close()
+
+	if err := logging.Close(); err != nil {
+		log.Debugf("Error closing log: %v", err)
+	}
 	os.Exit(0)
 }
 
 func doMain() error {
-	err := logging.Init()
-	if err != nil {
+	if err := logging.Init(); err != nil {
 		return err
 	}
 
 	// Schedule cleanup actions
 	handleSignals()
-	addExitFunc(func() { logging.Close() })
+	addExitFunc(func() {
+		if err := logging.Close(); err != nil {
+			log.Debugf("Error closing log: %v", err)
+		}
+	})
```

Comment:
This code to close logging and check the result appears twice (also at 102-105 above). Might be advantageous/cleaner to just move it to a separate function and use that instead?